deploy/setup/scripts/web-config.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+set -e
+
+echo "🔧 Starting web-based configuration wizard..."
+
+# --- Expected env vars ---
+DEPLOY_DIR="${DEPLOY_DIR:?DEPLOY_DIR not set}"
+LOG_PATH="${LOG_PATH:-/tmp/wbs-deploy-setup.log}"
+VERBOSE="${VERBOSE:-false}"
+LOCALHOST="${LOCALHOST:-false}"
+CLOUD_INIT="${CLOUD_INIT:-false}"
+CERT_EMAIL="${CERT_EMAIL:-wbs-setup@wikimedia.de}"
+
+# -- Script specific env vars --
+SETUP_PORT=8888
+SETUP_DIR="$DEPLOY_DIR/setup"
+CERTS_DIR="$SETUP_DIR/certs"
+VIEWS_DIR="$SETUP_DIR/views"
+
+# -- Setup logging --
+# shellcheck disable=SC1091
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/_logging.sh"
+
+generate_cert_for_setup_webserver() {
+  log "Generating TLS certificate for setup page..."
+
+  if [ "$LOCALHOST" = true ]; then
```

Comment:
Have you considered using Traefik for this? This could spare us the manual steps as well as use tech we already depend on anyway.

Reply:
Interesting approach 👍🏾